src/pybind/mgr/dashboard/controllers/monitor.py

```diff
@@ -14,7 +14,7 @@
 class Monitor(BaseController):
     @cherrypy.expose
     @cherrypy.tools.json_out()
-    def default(self):
+    def default(self, *_vpath, **_params):
```

Comment:
Is there a way to not require to add those parameters in all `default` methods?

Reply:
Pylint says; `W:464, 4: Parameters differ from overridden 'default' method (arguments-differ)`. In general, I think, it's OK to have consistent method arguments for derived classes.